modules/contentful-graphql/queries/static-page.graphql

```diff
@@ -0,0 +1,73 @@
+query StaticPage(
```

Comment:
This whole query is moved to be part of BrowseStaticPage, and actually never used right? 
Then let's get rid of it.